src/DroidRpc/client.py

```diff
@@ -32,14 +32,19 @@ class Client:
 
     batch_size = 400
 
-    def __init__(self, address: str = "guardian", port: str = "50065"):
+    def __init__(self,
+                 address: str = "guardian",
+                 port: str = "50065",
+                 batch_size: int = None):
```

Comment:
Why not put `batch_size: int = 400` instead of None? Then  you don't need the if statement below